test/1_cli.bats

```diff
@@ -217,24 +216,24 @@ teardown() {
   # report 1 - criticals - magnitude sort/lowest number first
   CRITICAL_FIRST_FIRST=$(jq -r .[0].scoring.critical[0].points <<<"${output}")
   CRITICAL_FIRST_SECOND=$(jq -r .[0].scoring.critical[1].points <<<"${output}")
-  (( CRITICAL_FIRST_FIRST <= CRITICAL_FIRST_SECOND ))
+  ((CRITICAL_FIRST_FIRST <= CRITICAL_FIRST_SECOND))
 
   # report 1 - advisories - magnitude sort/highest number first
   ADVISE_FIRST_FIRST=$(jq -r .[0].scoring.advise[0].points <<<"${output}")
   ADVISE_FIRST_SECOND=$(jq -r .[0].scoring.advise[1].points <<<"${output}")
   ADVISE_FIRST_THIRD=$(jq -r .[0].scoring.advise[2].points <<<"${output}")
-  (( ADVISE_FIRST_FIRST >= ADVISE_FIRST_SECOND >= ADVISE_FIRST_THIRD ))
+  ((ADVISE_FIRST_FIRST >= ADVISE_FIRST_SECOND >= ADVISE_FIRST_THIRD))
 
   # report 2 - criticals - magnitude sort/lowest number first
   CRITICAL_SECOND_FIRST=$(jq -r .[1].scoring.critical[0].points <<<"${output}")
   CRITICAL_SECOND_SECOND=$(jq -r .[1].scoring.critical[1].points <<<"${output}")
-  (( CRITICAL_SECOND_FIRST <= CRITICAL_SECOND_SECOND ))
+  ((CRITICAL_SECOND_FIRST <= CRITICAL_SECOND_SECOND))
 
   # report 2 - advisories - magnitude sort/highest number first
   ADVISE_SECOND_FIRST=$(jq -r .[1].scoring.advise[0].points <<<"${output}")
   ADVISE_SECOND_SECOND=$(jq -r .[1].scoring.advise[1].points <<<"${output}")
   ADVISE_SECOND_THIRD=$(jq -r .[1].scoring.advise[2].points <<<"${output}")
-  (( ADVISE_SECOND_FIRST >= ADVISE_SECOND_SECOND >= ADVISE_SECOND_THIRD ))
+  ((ADVISE_SECOND_FIRST >= ADVISE_SECOND_SECOND >= ADVISE_SECOND_THIRD))
```

Comment:
The bash styleguide we follow has spaces after `((` and before `))`

https://google.github.io/styleguide/shellguide.html#arithmetic

Reply:
@06kellyjac updated, FI, this is the result of `shfmt`

Reply:
$(( and (( ambiguity is not supported. Backtracking would complicate the parser and make streaming support via io.Reader impossible. The POSIX spec recommends to [space the operands](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_03) if $( ( is meant.
$ echo '$((foo); (bar))' | shfmt
1:1: reached ) without matching $(( with ))